README.md

```diff
@@ -22,19 +22,29 @@ pnpm add vue-bundle-renderer
 ### `createRenderer`
 
 ```ts
-import { createRenderer } from 'vue-bundle-renderer'
+import { createRenderer } from 'vue-bundle-renderer/runtime'
 
 declare function createRenderer(createApp:, renderOptions: RenderOptions)
 ```
 
-### `normalizeClientManifest`
+### `normalizeViteManifest`
```

Comment:
(or `defineRendererManifest` with `bundler: 'vite' | 'webpack'` to set the input type)

Reply:
fromViteManifest?

Reply:
`mapViteManifest`?

If we are running this at build time we also can have a shared function with bundler option as you suggest without any serious perf issue, so we can drop bundler time from name and just run `createRendererManifest`?

Reply:
Equally happy with `normalizeRendererManifest`?

Reply:
I love both `createRendererManifest` and `mapViteManifest` ideas. Having bundler-specific functions makes more sense to me because this reduces the complexity of the single function and clear usage. We anyway have two different packages that consume and what matters is output is of type RendererManifest. Either mapped from a bundler or directly defined.

Reply:
Are you suggesting we also expose the vite/webpack specific functions, or the opposite?

Reply:
To expose them (opposite to my second comment :D)